### PR TITLE
Partly document bag sources and header

### PR DESCRIPTION
**Pull request by LangeTreeDorpie**
This is my first ever PR for an opensource project and also my first try on a reverse engineering project. 

But don't be gentle and shoot me 😄 

 ## Renamed Files
 * `include/unk_0207D3B8.h` -> `include/bag.h`
 * `src/unk_0207D3B8.c` -> `src/bag.c`
 
 ## Renamed Structs and Fields
 * `UnkStruct_0207CDEC` -> `ItemSlot`
 * `UnkStruct_0207D3C0` -> `Bag`
   *  `unk_00` -> `items`
   *  `unk_294` -> `keyItems`
   *  `unk_35C` -> `TMsHMs`
   *  `unk_4EC` -> `mail`
   *  `unk_51C` -> `medicine`
   *  `unk_5BC` -> `berries`
   *  `unk_6BC` -> `balls`
   *  `unk_6F8` -> `battleItems`
   *  `unk_770` -> `registeredItem`
 
 ## Renamed Functions
 * `sub_0207D3D8` -> `Sav2_Bag_init`
 * `sub_0207D3EC` -> `Sav2_Bag_copy`
 * `sub_0207D3FC` -> `Bag_GetRegisteredItem`
 * `sub_0207D404` -> `Bag_SetRegisteredItem`
 * `sub_0207D40C` -> `Bag_GetItemPocket`
 * `sub_0207D4B4` -> `Pocket_GetItemSlotForAdd`
 
 ## New Enums and Typedefs
 None named/exported.
 
Got most of the struct/function names from the diamond repo

### CODE REVIEW

File: include/struct_defs/struct_0207CDEC.h [hunk 1]
```diff
@@ -4,6 +4,6 @@
 typedef struct {
```
Comment (Nomura-RH) [inline include/struct_defs/struct_0207CDEC.h:4]: Structs must be typedef'd, and have the same name as the typedef, i.e.

`typedef struct ItemSlot {
    u16 unk_00;
    u16 unk_02;
} ItemSlot;`

File: include/bag.h [hunk 2]
```diff
@@ -0,0 +1,42 @@
+#ifndef POKEPLATINUM_UNK_0207D3B8_H
```
Comment (Nomura-RH) [inline include/bag.h:1]: You forgot to change the header guard :)

File: include/bag.h [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+#define POKEPLATINUM_UNK_0207D3B8_H
+
+#include "struct_defs/struct_0207CDEC.h"
+#include "struct_decls/struct_0207D3C0_decl.h"
+#include "struct_decls/struct_0207D99C_decl.h"
+#include "struct_decls/struct_021C0794_decl.h"
+
+int bag(void);
```
Comment (Nomura-RH) [inline include/bag.h:9]: Bag_sizeof

**Review by Nomura-RH — CHANGES_REQUESTED** [review]
Hi! Sorry for taking this long to review, I have been busy these past weeks. This PR looks good, I left a couple comments about minor adjustments I'd like you to make, but great job overall! Contributions like these are greatly appreciated, and I hope you will do more in the future.

**Comment by PikalaxALT** [comment]
Is this PR abandoned?

**Comment by lhearachel** [comment]
Closing in favor of #229

**Pull request closed without merging**